src/entities/mellow/mellowLpRouter.ts

```diff
@@ -215,6 +219,22 @@ class MellowLpRouter {
     return getTokenInfo(this.readOnlyContracts.token.address).decimals;
   }
 
+  public get depositable(): boolean {
+    const latestMaturity = this.metadata.vaults.reduce(
+      (latest, vault) => Math.max(latest, vault.maturityTimestampMS),
+      0,
+    );
+    return !this.metadata.deprecated && Date.now().valueOf() + 48 * ONE_HOUR_IN_MS < latestMaturity;
```

Comment:
again the magic 48 :) lets move the entire thing to constatnts.ts, the 48*ONE_HOUR_IN_MS.